operator/config/crd/bases/csi.ibm.com_csiscaleoperators_cr.yaml

```diff
@@ -0,0 +1,167 @@
+---
+apiVersion: csi.ibm.com/v1
+kind: "CSIScaleOperator"
+metadata:
+  name: "ibm-spectrum-scale-csi"
+  namespace: "ibm-spectrum-scale-csi-driver"
+  labels:
+    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
+    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
+    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
+    release: ibm-spectrum-scale-csi-operator
+status: {}
+spec:
+
+# A passthrough option that distributes an imagePullSecrets array to the containers
+# generated by the csi scale operator. Please refer to official k8s documentation for
+# your environment for more details. https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+# ==================================================================================
+#  imagePullSecrets:
+#    - APullSecret
+#    - AnotherOptional
+
+# Below specifies the details of a SpectrumScale cluster configuration used by the
+# plugin. It can have multiple values.
+# ==================================================================================
+  clusters:
+    - id: "< Primary Cluster ID - WARNING - THIS IS A STRING NEEDS YAML QUOTES! >"
+      secrets: "secret1"
+      secureSslMode: false
+      primary:
+        primaryFs: "< Primary Filesystem >"
+#        primaryFset: "< Fileset in Primary Filesystem >"   # Optional - default:spectrum-scale-csi-volume-store
+#        inodeLimit: "< inode limit for Primary Fileset >" # Optional
+#        remoteCluster: "< Remote ClusterID >"            # Optional - This is only required if primaryFs is remote cluster's filesystem and this ID should have separate entry in Clusters map too.
+#      cacert:  "< Name of CA cert configmap for GUI >"   # Optional
+      restApi:
+        - guiHost: "< Primary cluster GUI IP/Hostname >"
+#
+# In the case we have multiple clusters, specify their configuration below.
+# ==================================================================================
+#    - id: "< Cluster ID >"
+#      secrets: "< Secret for Cluster >"
+#      secureSslMode: false
+#      restApi:
+#      - guiHost: "< Cluster GUI IP/Hostname >"
+#      cacert:  "< Name of CA cert configmap for GUI >"   # Optional
+
+# Attacher image name, in case we do not want to use default image.
```

Comment:
we do not intend to provide user a way to change the sidecar images

Reply:
Done